tests/specs/helpers/assertion-helpers.js

```diff
@@ -0,0 +1,20 @@
+function getMetricsFromResponse (response, isSupportability) {
```

Comment:
Is this really necessary anymore? We no longer send metrics in query parameters.

Reply:
Removed